internal/dataplane/kongstate/kongstate_test.go

```diff
@@ -788,6 +790,7 @@ func TestKongState_BuildPluginsCollisions(t *testing.T) {
 	for _, tt := range []struct {
 		name       string
 		in         []*kongv1.KongPlugin
+		kv         semver.Version
```

Comment:
can we rename it to `kongVersion`? I imagine other people in the future struggling to understand what `kv` is 😄

Reply:
done.